stream/stream_bluray.c

```diff
@@ -553,7 +553,7 @@ static bool check_bdmv(const char *path)
 
     char data[50] = {0};
 
-    fread(data, 50, 1, temp);
+    (void)fread(data, 50, 1, temp);
```

Comment:
I would prefer to actually check it, same as `stream_dvdnav` does. https://github.com/mpv-player/mpv/blob/6c59b0272b93e2ed4fb3842a0135e1b5e2c907f3/stream/stream_dvdnav.c#L182